apps/generic-issuance-client/src/pages/pipeline/PipelineDetailSection.tsx

```diff
@@ -44,120 +50,216 @@ export function PipelineDetailSection({
         </AccordionItem>
 
         {isAdminView && (
-          <AccordionItem>
-            <AccordionButton>
-              Tracing Links&nbsp;<Badge colorScheme="gray">Admin</Badge>
-            </AccordionButton>
-            <AccordionPanel>
-              <UnorderedList>
-                <ListItem>
-                  <PodLink
-                    isExternal={true}
-                    to={getLoadTraceHoneycombLinkForPipeline(pipeline.id)}
-                  >
-                    data load traces {getHoneycombQueryDurationStr()}
-                  </PodLink>
-                </ListItem>
-                <li>
-                  <PodLink
-                    isExternal={true}
-                    to={getAllHoneycombLinkForPipeline(pipeline.id)}
-                  >
-                    all traces related to this pipeline{" "}
-                    {getHoneycombQueryDurationStr()}
-                  </PodLink>
-                </li>
-              </UnorderedList>
-            </AccordionPanel>
-          </AccordionItem>
+          <>
+            {supportsManualTicketTable(pipeline) && isAdminView && (
+              <AccordionItem>
+                <AccordionButton>
+                  Add Manual Ticket&nbsp;<Badge colorScheme="gray">Admin</Badge>
+                </AccordionButton>
+                <AccordionPanel>
+                  <SectionContainer>
+                    <PipelineAddManualTicketSection
+                      pipeline={pipeline}
+                      isAdminView={isAdminView}
+                    />
+                  </SectionContainer>
+                </AccordionPanel>
+              </AccordionItem>
+            )}
+
+            {supportsAddingManualTickets(pipeline) && isAdminView && (
+              <AccordionItem>
+                <AccordionButton>
+                  Existing Manual Tickets&nbsp;
+                  <Badge colorScheme="gray">Admin</Badge>
+                </AccordionButton>
+                <AccordionPanel>
+                  <SectionContainer>
+                    <PipelineDisplayManualTicketsSection
+                      pipeline={pipeline}
+                      isAdminView={isAdminView}
+                    />
+                  </SectionContainer>
+                </AccordionPanel>
+              </AccordionItem>
+            )}
+            <AccordionItem>
+              <AccordionButton>
+                Tracing Links&nbsp;<Badge colorScheme="gray">Admin</Badge>
+              </AccordionButton>
+              <AccordionPanel>
+                <SectionContainer>
+                  <UnorderedList>
+                    <ListItem>
+                      <PodLink
+                        isExternal={true}
+                        to={getLoadTraceHoneycombLinkForPipeline(pipeline.id)}
+                      >
+                        data load traces {getHoneycombQueryDurationStr()}
+                      </PodLink>
+                    </ListItem>
+                    <li>
```

Comment:
Why is this a `<li>` and not a `<ListItem>` as above? (or vice versa)

Reply:
no reason; fixed